doc/nrf/ug_nrf52_gs.rst

```diff
@@ -0,0 +1,215 @@
+.. _ug_nrf52_gs:
+
+Getting started with nRF52 Series
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section gets you started with your nRF52 Series :term:`Development Kit (DK)` using the |NCS|.
+It tells you how to install the :ref:`peripheral_uart` sample and perform a quick test of your DK.
+
+If you have already set up your nRF52 Series DK and want to learn more, see the following documentation:
+
+* :ref:`ug_nrf52` for more advanced topics related to the nRF52 Series if you are already familiar with the |NCS|.
+* The :ref:`introductory documentation <getting_started>` for more information on the |NCS| and the development environment.
+
+.. _nrf52_gs_requirements:
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer and mobile device both have one of the supported operating systems.
+
+Hardware
+========
+
+* One of the following nRF52 Series development kits:
+
+  * nRF52840 DK
+  * nRF52833 DK
+  * nRF52 DK
+
+* Micro-USB 2.0 cable
+
+Software
+========
+
+On your computer, one of the following operating systems:
+
+* Microsoft Windows 10
+* macOS X, latest version
```

Comment:
This is the way it is expressed in the other Getting started sections, though it might be changing now that we've added more information in the requirements section of the documentation.